tests/test_cli.py

```diff
@@ -39,7 +40,9 @@ def test_main_with_nonexistent_config_file():
     type(result.exception) == FileNotFoundError
 
 
-def test_controller_plans():
+@patch("requests.get")
+def test_controller_plans(mock_requests: Mock):
```

Comment:
should this test be renamed? It's more testing the failure case than the controller plans method/endpoint

Reply:
Sure, good idea. Have renamed this now.